src/js/components/ItemVolumeTable.js

```diff
@@ -0,0 +1,192 @@
+import classNames from 'classnames';
```

Comment:
I'm not sure about the naming... I'd suggest something like `ServiceVolumeTable`